pkg/apiserver/controller/cluster/cluster.go

```diff
@@ -69,6 +70,34 @@ type ListClusterRequest struct {
 	ClusterStatus   string   `json:"clusterStatus"`
 }
 
+type ListClusterByLabelRequest struct {
```

Comment:
It might be better to rename `ListClusterByLabelRequest` to `ListClusterResourcesRequest`